roipoly/roipoly.py

```diff
@@ -152,6 +152,25 @@ def display_mean(self, image, **textkwargs):
                  string, color=self.color,
                  bbox=dict(facecolor='w', alpha=0.6), **textkwargs)
 
+    def get_roi_coordinates(self, image):
```

Comment:
in fact, it would be nice if one could also get back the coordinates in the space of the image! then, you would need `image`. 

I propose you do
```
def get_roi_coordinates(self, image=None)
```

and then you do something like

```
if image is not None:
    # Return ROI coordinates in the space of `image`
    ...
else:
    roi_coordinates = list(zip(self.x, self.y))
    ...
return roi_coordinates   
```

For returning the coordinates in the space of the image, see the `get_mask()` function.

If you implement it that way, you need to state in the docstring in which space the coordinates are returned, when the argument `image` is supplied or not.

Reply:
Here by 'ROI coordinates in the space of image' are you referring to the entire list of coordinates within the ROI? Wouldn't that essentially be the coordinates which are marked as True in mask.. Also I didn't quite understand what you meant by 'in which _space_ the coordinates are returned'. Could you please elaborate on that as well?

Reply:
Hey, sorry, my misunderstanding.
You can just ignore the last comment (however, I think you still don't need `image` here)

Reply:
Hi, You're definitely right on that! Will make that change